src/monika/Assignment30_FindSecondMaxNo/FindSecondMaxNoInArray.java

```diff
@@ -0,0 +1,30 @@
+package monika.Assignment30_FindSecondMaxNo;
+/*Assignment - 30 : 4th May'2021
+WAP to find Second max from given array
+int[] arr = {10,33,43,55,97,11,3};
+ouput : 55 */
+
+public class FindSecondMaxNoInArray {
+	
+	int getSecondMaxNum(int[] input) {
+		for (int i = 0; i < input.length; i++) {
+			int temp = 0;
+			for (int j = i + 1; j <= input.length; j++) {
```

Comment:
You can handle this with single loop as well

Shashank - https://github.com/KrishnaKTechnocredits/JAVATechnoMar2021/pull/1560/files